tests/data/minfraud/full-request.json

```diff
@@ -46,6 +46,7 @@
     "decline_code": "invalid number"
   },
   "credit_card": {
```

Comment:
One optional suggestion is to have a request containing country code but no issuer_id! In that case we should not get any exceptions.